scanner/src/main/kotlin/Main.kt

```diff
@@ -26,7 +26,10 @@ import com.beust.jcommander.JCommander
 import com.beust.jcommander.Parameter
 import com.beust.jcommander.ParameterException
 
+import com.fasterxml.jackson.databind.JsonMappingException
```

Comment:
Commit message: Comma after "Before". And maybe say "Previously" instead?

Reply:
Done